docs/developers-guide.md

```diff
@@ -24,24 +24,27 @@ If you have problems with your development environment, make sure that you are n
 
 These are the set of tools which are required in order to complete any build of the Metabase code. Follow the links to download and install them on your own before continuing.
 
-1. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - latest LTS version of JDK - more about [Java versions](./operations-guide/java-versions.md)
-2. [Node.js (http://nodejs.org/)](http://nodejs.org/) - latest LTS release
-3. [Yarn package manager for Node.js](https://yarnpkg.com/) - latest release of version 1.x
-4. [Leiningen (http://leiningen.org/)](http://leiningen.org/) - latest release
+1. [Clojure (ttps://clojure.org)](https://clojure.org/guides/getting_started) - install the latest release by following the guide depending on your OS
+2. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - you need to install JDK 8 (to build) and JDK 11 (to run Metabase) - more about [Java versions](./operations-guide/java-versions.md)
```

Comment:
To make it simple, will the use of *only* JDK 8 work here? Mentioning two versions of JDK can be confusing.

Reply:
Mmm let me check, I'll wipe JDK 11 on my machine and see if it works. The guide right now is with JDK 11, I included JDK 8 since the build process asks for JDK 8 explicitly

Reply:
yup, builds with JDK 8 only, no need to ask for JDK 11, do we go forward and and wipe the JDK 11 package? why it was there in the first place?

Reply:
I don't remember the details, but something with gettext and Java 11 interplay. In all case, safe to remove the notion of JDK 11 for now (until we will be able to resolve the issue).

Reply:
i18n utils: https://github.com/metabase/metabase/blob/master/Dockerfile#L5

Reply:
thanks @flamber ! so we remove JDK 11 as a dependency in the dev guide then?

Reply:
It's probably fine with just JDK 8 - should be possible to build with JDK 11 too (I know I have), but problematic on Alpine.

Reply:
great! I just did the edit and will be resolving this convo